perfkitbenchmarker/providers/alicloud/util.py

```diff
@@ -109,11 +108,24 @@ def AddPubKeyToHost(host_ip, password, keyfile, username):
       public_key = f.read()
   ssh = paramiko.SSHClient()
   ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+
+  # this section is needed in case a corporate proxy is in play
+  proxy = None
+  ssh_config_file = os.path.expanduser("~/.ssh/config")
```

Comment:
Can this be a flag with the other ssh related ones in https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/9915b334621acc8c262fb47ff38846a873d54531/perfkitbenchmarker/vm_util.py#L92

Or prefixed with paramiko

Reply:
@dlott I added another commit removing the paramiko dependency altogether. Not sure why the key exchange was initially implemented through paramiko, but I didn't see a particular need for it. It was often hiding connection errors and made things hard to debug. The current approach, using ssh key pairs, is more robust and aligned with the implementation for other cloud providers.